packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -347,7 +347,7 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
     let shouldComponentUpdate = false;
 
     // Update if the page stops scrolling
-    if (!newState.isScrolling && this.state.isScrolling) {
+    if (!newState.isScrolling && this.state.isScrolling && !this.props.ignoreScrollingState) {
```

Comment:
no need to add this condition here since `isScrolling` won't change

Reply:
Fair enough.